Dockerfile

```diff
@@ -1,10 +1,17 @@
-FROM node:16-slim as node-build
+FROM node:19-slim as node-build
 
 ARG NPM_CONFIG__AUTH
 ARG NPM_CONFIG_REGISTRY=https://workivaeast.jfrog.io/workivaeast/api/npm/npm-prod/
 ARG NPM_CONFIG_ALWAYS_AUTH=true
 ARG GIT_TAG
 
+RUN reg=$(echo "$NPM_CONFIG_REGISTRY" | cut -d ":" -f 2) && \
```

Comment:
If this is what's required, it seems like we should report that to #support-build to let them know how tedious this is so they can, e.g., add an `NPMRC_CONTENT` env var that has the full contents of this file (ala `KNOWN_HOSTS_CONTENT`).